python/computer_use_agent/.gitignore

```diff
@@ -0,0 +1,19 @@
+# TeamsFx files
+env/.env.*.user
+env/.env.local
```

Comment:
do we want to push the non-filled /env folder up? we usually did this for teams-ai samples, so devs could reference what they look like & fill in w/ pre-deployed vars